devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -430,6 +433,11 @@ private void addToClassPaths(StringBuilder classPathManifest, DevModeContext cla
             throw new RuntimeException(e);
         }
         String path = uri.getRawPath();
+        if (PropertyUtils.isWindows()) {
+            if (path.length() > 2 && Character.isLetter(path.charAt(0)) && path.charAt(1) == ':') {
+                path = "/" + path;
+            }
+        }
```

Comment:
Wondering if we should have an util for that as we have the exact same dark magic code in 2 places? Not a blocker, just a though.

Reply:
Maybe at some point.  There are probably a lot of things that can be cleaned up in this code.